OfficeAddin/05 Using modern JavaScript/Demos/02 Office Add-in using Angular/README.md

```diff
@@ -340,433 +289,450 @@ In this section, you will finish developing the Office Add-in using Angular and
             </div>
         </div>
     </div>
-    ````
+    ```
 
 1. Next, open **src/app/app.component.ts** and update it as follows.
 
-    ````typescript
+    ```typescript
```

Comment:
global comment - missing license and copywrite information.

Reply:
All code files generated will include the license & copyright information for generated files where the Office generator created the files. However, the latest version does not put these comments at the top of all files anymore.